api/v1alpha1/docker_clusterconfig_types.go

```diff
@@ -21,11 +25,21 @@ type DockerClusterConfig struct {
 // DockerClusterConfigSpec defines the desired state of DockerClusterConfig.
 type DockerClusterConfigSpec struct {
 	GenericClusterConfig `json:",inline"`
+
+	//+optional
+	CustomImage *string `json:"customImage,omitempty"`
```

Comment:
Do we want this as `OCIImage`?
```suggestion
	CustomImage *OCIImage `json:"customImage,omitempty"`
```

Reply:
Oops! Thank you!

Reply:
Thanks!